wordclock_interfaces/templates/app.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Word Clock</title>
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel='stylesheet prefetch' href='https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons'>
+    <link rel='stylesheet prefetch' href='https://unpkg.com/vuetify/dist/vuetify.min.css'>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+    <script src='https://unpkg.com/vue/dist/vue.min.js'></script>
+    <script src='https://unpkg.com/vuetify/dist/vuetify.js'></script>
+    <script src='https://unpkg.com/babel-polyfill/dist/polyfill.min.js'></script>
+    <script src="https://unpkg.com/@radial-color-picker/vue-color-picker/dist/vue-color-picker.min.js"></script>
+    <link rel="stylesheet" href="static/vue-color-picker.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css">
+</head>
+
+<body>
+
+    <div id="app">
+        <v-app id="inspire" dark>
+            <v-content>
+                <v-container fluid fill-height>
+                    <v-layout align-center justify-center>
+                        <v-flex xs12 sm8 md4>
+                            <v-tabs dark color="primary" class="elevation-12">
+                                <v-tab ripple height="100%">
+                                    Control
+                                </v-tab>
+                                <v-tab ripple>
+                                    Colours
+                                </v-tab>
```

Comment:
How about adding another tab with some links to the project-page and API. In this case, the app would become ready to replace the current index.html... ;)

Reply:
Links would be:
* The project page: https://github.com/bk1285/rpi_wordclock
* The API: wordclock-ip//api

Reply:
Working on this one....

Reply:
About page added.
See if you can find it :-)

Reply:
Found it! Nice! :+1: